File: JavaScript/Iniciante/1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

linha1 = lines.shift().split(" ")
linha2 = lines.shift().split(" ")

x1 = parseFloat(linha1[0])
y1 = parseFloat(linha1[1])

x2 = parseFloat(linha2[0])
y2 = parseFloat(linha2[1])

distancia = Math.sqrt(Math.pow(x2-x1, 2) + Math.pow(y2-y1, 2))

console.log(`${distancia.toFixed(4)}`)

File: C++/Iniciante/1015.cpp
#include <stdio.h>
#include <iostream>
#include <cmath>

int main() {

  double x1, y1, x2, y2, distancia;
  std::cin >> x1 >> y1;
  std::cin >> x2 >> y2;

  distancia = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));

  printf("%.4lf\n", distancia);
  
}
